storage/remote/remote.proto

```diff
@@ -40,7 +40,8 @@ message ReadRequest {
 }
 
 message ReadResponse {
-  repeated TimeSeries timeseries = 1;
+  // In same order as the request's queries.
+  repeated QueryResponse responses = 1;
```

Comment:
How about `QueryResult`, since the `Response` suffix is what we always use for the overall protobuf request/response message.